### PR TITLE
🏗 🐛 Fix bento publish flow

### PR DESCRIPTION
**Pull request by rileyajones**
The Bento publish flow has been failing for about 2 weeks for 2 different reasons...
[Core publishing issue logs](https://github.com/ampproject/amphtml/runs/5431875974?check_suite_focus=true)
[Bento components directory issue logs](https://github.com/ampproject/amphtml/runs/5301120632?check_suite_focus=true)

Core is not being published because the directory is changed prior to the script determining the version being called.

Components are not being published because the publish script assumes all packages are under the `extensions` directory.

#37737

### CODE REVIEW

File: .github/workflows/publish-npm-packages.yml [hunk 1]
```diff
@@ -125,13 +127,15 @@ jobs:
       - name: Tag package as latest for Stable releases
         if: github.event.inputs.tag == 'latest'
         run: |
-          NPM_PACKAGE=$(jq -r .name ./extensions/${{ matrix.extension }}/${{ matrix.version }}/package.json)
-          NPM_VERSION=$(jq -r .version ./extensions/${{ matrix.extension }}/${{ matrix.version }}/package.json)
+          cd $(node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js ${{ matrix.extension }} ${{ matrix.version }})
```
Comment (kvchari) [inline .github/workflows/publish-npm-packages.yml:130]: wait should this be?
```suggestion
          cd $(node ${{ env.SCRIPTS_DIR }}/get-package-dir.js
```

**Review by kvchari — COMMENTED** [review]
Can't believe I missed this, thanks for catching! 1 comment

File: .github/workflows/publish-npm-packages.yml [hunk 2]
```diff
@@ -69,9 +69,10 @@ jobs:
         run: |
           npm ci
           node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js
-          cd ./src/bento/core
           CORE_VERSION=$(node ${{ env.SCRIPTS_DIR }}/get-core-version.js ${{ github.event.inputs.amp-version }})
```
Comment (kvchari) [inline .github/workflows/publish-npm-packages.yml:72]: my bad I think I was assuming this was an absolute path when I wrote this, but just realized it isn't. 

Also don't think the `cd -` is strictly necessary. I tested locally and it seemed like each step of a job is reinitialized to the root of the repo

Comment (kvchari) [reply .github/workflows/publish-npm-packages.yml:72]: np! by locally, I meant i just ran (modifed) actions in my fork and verified that changing directories in [one step](https://github.com/kvchari/amphtml/runs/5383481179?check_suite_focus=true#step:6:12) doesn't affect the directory that [another step](https://github.com/kvchari/amphtml/runs/5383481179?check_suite_focus=true#step:7:10) starts in

File: .github/workflows/publish-npm-packages.yml [hunk 3]
```diff
@@ -119,19 +121,24 @@ jobs:
           node ${{ env.SCRIPTS_DIR }}/write-package-files.js ${{ matrix.extension }} ${{ github.event.inputs.amp-version }} ${{ matrix.version }}
       - name: Publish package for Nightly releases
         if: github.event.inputs.tag != 'latest'
-        run: npm publish ./extensions/${{ matrix.extension }}/${{ matrix.version }} --access public --tag ${{ github.event.inputs.tag }}
+        run: |
+          cd $(node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js ${{ matrix.extension }} ${{ matrix.version }})
```
Comment (kvchari) [inline .github/workflows/publish-npm-packages.yml:125]:
```suggestion
          cd $(node ${{ env.SCRIPTS_DIR }}/get-package-dir.js
```

**Comment by kvchari** [comment]
will you/can you kick off the npm publish workflow to confirm that it all works correctly?

**Comment by rileyajones** [comment]
> will you/can you kick off the npm publish workflow to confirm that it all works correctly?

Seems like I don't have permission
https://github.com/ampproject/amphtml/actions/runs/1965853127

**Pull request merged**